src/natcap/invest/recreation/recmodel_client.py

```diff
@@ -1458,8 +1458,8 @@ def _validate_same_projection(base_vector_path, table_path):
     """
     # This will load the table as a list paths which we can iterate through 
     # without bothering the rest of the table structure
-    data_paths = pandas.read_csv(
-        table_path, squeeze=True, usecols=['path']).tolist()
+    data_paths = utils.read_csv_to_dataframe(
+        table_path, squeeze=True)[['path']].tolist()
```

Comment:
Sorry, this is in part due to my own naivete with `pandas`: what are the tradeoffs to consider between the `usecols` approach and the indexing (`[['path']]`) here?

Reply:
`usecols` will only read in the given columns into the dataframe, so if the column name had extra whitespace e.g. 'path ', I don't think it would work. So rather than selecting the column when the CSV is read, I'm first reading in all the columns, removing extra whitespace, and then indexing by 'path'.

Reply:
Ah,  I see ... very interesting.